mailbox/hbase/src/main/java/org/apache/james/mailbox/hbase/mail/HBaseMailboxMapper.java

```diff
@@ -388,4 +390,16 @@ public void deleteAllMailboxes() {
     public void updateACL(Mailbox<HBaseId> mailbox, MailboxACL.MailboxACLCommand mailboxACLCommand) throws MailboxException {
         mailbox.setACL(mailbox.getACL().apply(mailboxACLCommand));
     }
+
+    @Override
+    public void setMetadata(Mailbox<HBaseId> mailbox, MailboxAnnotationCommand mailboxAnnoCommand)
```

Comment:
the capability won't be supported for HBase, you don't have to implement these methods I guess